app/components/IndividualStudent.js

```diff
@@ -0,0 +1,42 @@
+import React, { Component } from 'react';
+import { connect } from 'react-redux';
+import store from '../store.jsx'
+import Navbar from './Navbar'
+import {getStudentThunkCreator} from '../reducers/index.jsx'
+
+
+class getStudent extends Component{
```

Comment:
Convention dictates that javascript classes are written in `PascalCase` (sometimes referred to as `CamelCase`) with the first letter in uppercase.